Project Template/README.md

```diff
@@ -0,0 +1,49 @@
+# VMware Project Template in Visual Studio for .NET Automation SDK
+## Table of Contents
```

Comment:
We don't need such detailed instructions. Please keep it short to include only the required steps.